pctiler/pctiler/main.py

```diff
@@ -84,12 +85,20 @@
 
 app.include_router(health.health_router, tags=["Liveliness/Readiness"])
 
-app.add_exception_handler(Exception, http_exception_handler)
 add_timeout(app, settings.request_timeout)
 add_exception_handlers(app, DEFAULT_STATUS_CODES)
 add_exception_handlers(app, MOSAIC_STATUS_CODES)
 
 
+@app.exception_handler(HTTPException)
+async def http_exception_handler(
+    request: Request, exc: HTTPException
+) -> PlainTextResponse:
+    return PlainTextResponse(
```

Comment:
I think the nice thing about the previous use of `http_exception_handler` was that we'd log the stack trace. I think we should keep that behavior here, if it isn't preserved elsewhere. Also, is there a consequence to removing the handler from `Exception` and adding it to `HTTPException`? I'm curious if that response test that Tom asked about earlier in this PR still behaves the same?

Reply:
yeah... I think we lose global handling if we go with just HttpException. I can restore back to the handler 
```    
if isinstance(exc, HTTPException):
        raise
    else:
        logger.exception(
            "Exception when handling request",
            extra=get_custom_dimensions({"stackTrace": f"{exc}"}, request),
        )
        raise
```